motioneye/motionctl.py

```diff
@@ -37,6 +38,12 @@
 _motion_binary_cache = None
 _motion_detected = {}
 
+# select range function call based on Python version
+def _rangeOptim(x):
+    if sys.version[0] == 3:
+        return range(x)
+    else:
+        return xrange(x)
```

Comment:
This change is not required anymore since we have a dedicated `python2` branch now and a fix in `dev` already.